luigi/batch_notifier.py

```diff
@@ -108,21 +113,21 @@ def _format_tasks(self, tasks):
             return six.u('- {}').format(six.u('\n  ').join(lines))
 
     def _owners(self, owners):
-        return self._default_owner | set(owners or ())
+        return self._default_owner | set(owners)
 
-    def add_failure(self, task_name, family, unbatched_args, expl, owners=None):
+    def add_failure(self, task_name, family, unbatched_args, expl, owners):
```

Comment:
Ok. Just out of curiosity, is it still an anti-pattern to have an immutable object like `tuple()`?